### DISCUSSION TITLE
What is the relationship between ragflow and infinity?

### DISCUSSION THREAD

**Discussion by caishaojin** [infiniflow/infinity#1689] [Q&A]
What is the relationship between ragflow and infinity?

**Comment by JinHai-CN** [comment]
Infinity will be the database used by RAGFlow. So the current plan is that Infinity will be integrated into RAGFlow after the 0.3.0 release.

**Comment by JinHai-CN** [comment]
Infinity is integrated into RAGFlow, you may check PR: https://github.com/infiniflow/ragflow/pull/2894
